pkgs/development/python-modules/managesieve/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, pytestrunner
+, pytest
+}:
+
+with stdenv.lib;
+buildPythonPackage rec {
+  pname = "managesieve";
+  version = "0.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "ee70e298e9b68eb81f93d52a1320a034fdc182f3927fdd551836fc93b0ed2c5f";
+  };
+
+  doCheck = true;
+
+  checkInputs = [ pytestrunner pytest ];
+  propagatedBuildInputs = [ ];
+
+  meta = with lib; {
+    description = "ManageSieve client library for remotely managing Sieve scripts";
+    homepage    = "https://managesieve.readthedocs.io/";
+    license     = [ licenses.gpl3 licenses.psfl ];
```

Comment:
maybe add a comment as to why there are 2 licenses:
Python Software Foundation License for the module, GPL v3 for sieveshell and test suite.